deps/openblas.mk

```diff
@@ -31,6 +31,7 @@ endif
 endif
 
 # 64-bit BLAS interface
+$(error USE_BLAS64: $(USE_BLAS64))
```

Comment:
Is this intentional?

Reply:
Yeah I can no longer build with OpenBLAS from source because of this

Reply:
Oh, sorry; that was some debugging code left in...

Reply:
https://github.com/JuliaLang/julia/pull/56127